pkg/cli/admin/inspect/util.go

```diff
@@ -30,15 +31,35 @@ import (
 
 // resourceContext is used to keep track of previously seen objects
 type resourceContext struct {
-	visited sets.String
+	lock sync.Mutex
```

Comment:
is it possible to use RWMutex ?

Reply:
It's possible, but I don't think we'd benefit that much from it.